### PR TITLE
Move `zive` to devDependencies

### PR DESCRIPTION
**Pull request by qm3ster**
It is only used for compatability testing,
so no need to download it for production use.
Also updated other dependencies.
Didn't update to `debug@4` to maintain node4 compatability
Also removed some unused imports in some files

### CODE REVIEW

File: lib/components/query.js [hunk 1]
```diff
@@ -91,27 +91,27 @@ querie.endpoint = function (nwkAddr, epId, callback) {
...
 
-    querie.device(ieeeAddr, nwkAddr).then(function (devInfo) {
+    query.device(ieeeAddr, nwkAddr).then(function (devInfo) {
         fullDev = devInfo;
 
         _.forEach(fullDev.epList, function (epId) {
-            var epQuery = {func: querie.endpoint, nwkAddr: nwkAddr, epId: epId};
-            epQueries.push(epQuery);
+            var epQuery = {func: query.endpoint, nwkAddr: nwkAddr, epId: epId};
+            epquerys.push(epQuery);
```
Comment (Koenkk) [inline lib/components/query.js]: querys should be queries (https://www.dictionary.com/browse/queries)

File: lib/components/af.js [hunk 2]
```diff
@@ -66,10 +66,10 @@ af.send = function (srcEp, dstEp, cId, rawPayload, opt, callback) {
 
     areqTimeout = opt.hasOwnProperty('timeout') ? opt.timeout : undefined;
 
-    senderEp = srcEp.isLocal() ? srcEp : controller.getCoord().getDelegator(profId);
+    senderEp = srcEp.isLocal() ? srcEp : controller._coord.getDelegator(profId);
```
Comment (Koenkk) [inline lib/components/af.js:69]: In javascript it's a convention to not use properties with `_` externally. Either we should put back `getCoord` or we should rename `_coord` to `coord`.

Comment (qm3ster) [reply lib/components/af.js:69]: It's an intentional smell.
They claim it's private, but allow getting it directly.
I made the minimum change that didn't require me making the call of making it public.
Converting the projects to TypeScript which makes a distinction between `private` and `readonly` (and generally honors them) will bring more clarity.

File: lib/components/query.js (hunk 1 above)
Comment (qm3ster) [reply lib/components/query.js]: Totally agreed, will fix.

Comment (qm3ster) [reply lib/components/query.js]: Donezo.

**Comment by Koenkk** [comment]
Thanks!

**Pull request merged**